lib/trento_web/controllers/fallback_controller.ex

```diff
@@ -3,59 +3,38 @@ defmodule TrentoWeb.FallbackController do
 
   alias TrentoWeb.ErrorView
 
-  def call(conn, {:error, {:bad_request, errors = %{}}}) do
-    conn
-    |> put_status(:bad_request)
-    |> render_errors(errors)
-  end
-
-  def call(conn, {:error, {:bad_request, reason}}) do
-    conn
-    |> put_status(:bad_request)
-    |> render_error(reason)
-  end
-
-  def call(conn, {:error, {:unprocessable_entity, reason}}) do
+  def call(conn, {:error, {:unprocessable_entity, error}}) do
```

Comment:
What about the `bad_request` now?
Is it handled by `phoenix` itself?

Reply:
yes